core/src/crds_gossip_pull.rs

```diff
@@ -390,7 +390,8 @@ impl CrdsGossipPull {
         let past = now.saturating_sub(msg_timeout);
         let recent: Vec<_> = filters
             .iter()
-            .filter(|(caller, _)| caller.wallclock() < future && caller.wallclock() >= past)
+            .enumerate()
```

Comment:
doesn't seem like we need the enumerate here if the index is unused.

Reply:
@sakridge I think it's carried through to https://github.com/solana-labs/solana/pull/11734/files#diff-8b2c295066af153b1108d9db32997509R412

Reply:
yes it is carried further down to line 412 and used in line 425

Reply:
ah, I see